### ISSUE TITLE
can I change the email sender in our newsletter settings? 

### ISSUE THREAD

**Issue by davidmooreppf** [yescomment/sl-sludgeline#21]
Can I change the below from noreply@ to hello@ ? 

<img width="384" alt="Screenshot 2024-05-30 at 6 18 41 PM" src="https://github.com/yescomment/sl-sludgeline/assets/116819/7f4968db-a589-4aa2-9caa-cbd020775f19">

It replies to david@ just fine, as expected, but far better for it to display hello instead of noreply.

**Comment by unitof** [comment]
Yeah, requires we set you up a custom sending domain. Couple DNS records.
